scripts/build_images.sh

```diff
@@ -15,11 +15,44 @@
 
 set -ex
 
-docker build -t php-nginx php-nginx
-docker build -t php56 testapps/php56
-docker build -t php56_custom testapps/php56_custom
-docker build -t php70 testapps/php70
-docker build -t php56_70 testapps/php56_70
-docker build -t php70_custom testapps/php70_custom
-docker build -t php56_nginx_conf testapps/php56_nginx_conf
-docker build -t php56_custom_configs testapps/php56_custom_configs
+TAG="rc"
```

Comment:
release candidate? Maybe we should make this something more clear? This is not familiar to me in this context.